### DISCUSSION TITLE
z-index doesn't work when using popover.panel

### DISCUSSION THREAD

**Discussion by ozdv** [tailwindlabs/headlessui#2260] [Help]
Hello,

Project using react 18.2, headlessui/react 1.7, tailwind v3.2.

The issue I'm experiencing is when creating a popover menu to the right side of my navbar, it opens, however the content on the currentpage overlaps the menu.

My popover.panel classname is: 

```
<Popover.Panel className="absolute top-[-275px] left-[130px] z-10 -translate-x-1/2 transform px-2 sm:px-0 min-w-full">
```
and it looks like this:

![image](https://user-images.githubusercontent.com/7267982/217329070-5d6befc3-2a99-4ab1-9195-590a51e67c67.png)

Sorry if I missed any crucial detail; first time making discussion on git.  Thanks!

**Comment by matagaralph** [comment]
Have you tried using a higher z index value like z-40 or 50. ??

**Comment by nayan-7span** [comment]
Same issue is here. z-index is not affecting to the popover-panel.
